### DISCUSSION TITLE
Automatic figure caption numbering in HTML?

### DISCUSSION THREAD

**Discussion by elizabeth-hirsch** [quarto-dev/quarto-cli#10122] [Q&A]
### Description

I have a Quarto doc that renders to HTML and would like the figures (which are generated using R code) to each have a number before the caption. For example:

Figure 1. A plot displaying xyz

In RMarkdown the figures are numbered automatically, and it appears that when rendering to PDF using Quarto, the same automatic numbering is applied. However when I render to HTML only the caption is printed. Is there a way to add an automatic "Figure 1", "Figure 2", etc under figures in Quarto website?

The syntax I've tried using in code chunks is, for example:

```{r}
#| echo: false
#| results: asis
#| message: false
#| warning: false
#| label: tyee-plots
#| fig-cap: "Daily (top) and cumulative (bottom) estimated escapement at the Tyee fishery."

ggarrange(plot1,plot2,align="v",ncol=1)

```

Thank you!

**Comment by cderv** [comment]
The figure needs to have cross-reference enabled by using the right prefix. It should be all clear from our doc at: https://quarto.org/docs/authoring/cross-references.html

**Reply by mcanouil** [reply]
(I corrected a typo, it's prefix, not suffix. Weirdly I could while I can't original posts)
